test/unit/domain/wallets/create-wallet.spec.ts

```diff
@@ -0,0 +1,7 @@
+describe("wallet creation and update", () => {
+  it("creates a new wallet", async () => {
+    // const apikey = await addApiKeyForAccount()
+    // expect(isValidEncoding(apikey.key, "hex")).toBe(true)
+    // expect(isValidEncoding(apikey.secret, "base64")).toBe(true)
+  })
+})
```

Comment:
if ti is just a commented test please remove the file

Reply:
done